### ISSUE TITLE
Vision Assistant Pro not loading on NVDA 2024.4.2

### ISSUE THREAD

**Issue by gutorniero** [mahmoodhozhabri/VisionAssistantPro#77]
Hello Mahmood,
I’m having a problem with the Vision Assistant Pro add-on when using NVDA version 2024.4.2.
When NVDA starts and tries to load Vision Assistant Pro, it fails and the add-on doesn’t work. In the NVDA log, I get the following error:
ERROR - globalPluginHandler.listPlugins:
Error importing global plugin visionAssistant
AttributeError: 'NoneType' object has no attribute 'loader'
I don’t have technical knowledge, but it seems that something goes wrong while the add-on is loading, and because of that Vision Assistant Pro is not started.
I saw in the NVDA Add-on Store and in public announcements that Vision Assistant Pro was tested up to NVDA 2025.3.1, so I expected it to work with NVDA 2024.4.2 as well, but the error still happens.
Could you please let me know:
1. 
If this problem is already known?
2. 
If there is any workaround or fix I can try?
3. 
If a new version fixing this issue is planned?
Thank you very much for your work and your help!

**Comment by gutorniero** [comment]
Thank you very much for your attention and for the update. Do you have an
estimate of when the new version will be released?

Em qua., 24 de dez. de 2025 às 04:29, mahmoodhozhabri <
***@***.***> escreveu:

**Comment by mahmoodhozhabri** [comment]
The new version is ready. I am waiting for the contributors to provide the translation.
